src/components/error-dialog/index.tsx

```diff
@@ -9,6 +9,11 @@ import './styles.scss';
 
 const cn = bemClassName('error-dialog');
 
+export type ErrorDialogContent = {
```

Comment:
These will be used in the error dialog in a follow up PR. Keeping this PR separate from those changes.